source/Ship.cpp

```diff
@@ -4401,9 +4401,19 @@ double Ship::CalculateDeterrence() const
 			const Outfit *weapon = hardpoint.GetOutfit();
 			if(weapon->Ammo() && weapon->AmmoUsage() && !OutfitCount(weapon->Ammo()))
 				continue;
-			double strength = weapon->ShieldDamage() + weapon->HullDamage()
-				+ (weapon->RelativeShieldDamage() * attributes.Get("shields"))
-				+ (weapon->RelativeHullDamage() * attributes.Get("hull"));
+			// Ignore disabled and asteroid damages because they don't apply here.
+			// Ignore fuel and slowing damage because they don't affect combat ability.
+			// Over time effects are per frame but dont affect instantaneously so only multiply by 30.
```

Comment:
What's the basis for this?
Applying 10 discharge to a ship will result in the loss of 1000 shields over the course of the discharge's lifetime.
Why does the fact that it is spread out sufficient to make it only as deterring as 300 shield damage?
Why is disruption being multiplied by the same value? It's quite different a mechanic.

Reply:
~~because I dont truly understand all of these and thought it'd be a basis to improve upon since the base formula sucks so it cant be much worse~~
1000? damn that's a lot, but how long is said lifetime?